src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  margin-top: 20px;
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.search-bar-big {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  display: flex;
+  align-items: center;
+}
+
+.search-bar-big__icon {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.search-bar-big__input {
+  box-sizing: border-box;
```

Comment:
- 'box-sizing: border-box' is duplicated
 - better to apply box-sizing to every elements `*{ box-sizing: border-box }`